binhollc/SupernovaSDK-examples
# BIN-1012-Navadian

Add the SPI protocol API example
# Resolves

- https://focusuy.atlassian.net/browse/BIN-1012

# Changes

- Creates a notebook section for the Supernova SDK tests to adapt to the structure already created in BIN-991
- Adds the SPI protocol API example
- This example includes tests of the different commands related to the Supernova SPI module to show the feature's capabilities. It also includes an example of the Supernova interfacing with the Adafruit SPI FRAM, performing a write and read operation.

# How to test

- To test the commands, connect the breakout board to the Supernova and the logic analyzer in the correspondent SPI signals to verify the correct operation
- Follow step by step the explanation in the SPI-Protocol-Supernova-API.ipynb juypter notebook
- In the **Example with FRAM memory MB85RS64V** section connect the signals from the breakout board to its correspondent signals in the FRAM memory
- Execute the rest of the cells of the SPI-Protocol-Supernova-API.ipynb juypter notebook
